code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -17,6 +17,11 @@
 	var/limit = 10
 	var/speed = 1
 	var/list/holdingitems
+	///What objects will this allow for you to just insert and grind?
+	var/static/list/allowed_grindables = typecacheof(list(
+		/obj/item/food,
+		/obj/item/reagent_containers,
```

Comment:
why should you be able to grind beakers

Reply:
I could probably axe reagent containers if #55160 goes through, so I could totally wait for that to go through before this is ready.